contentcuration/contentcuration/frontend/channelEdit/translator.js

```diff
@@ -11,9 +11,25 @@ const MESSAGES = {
   errorQuestionRequired: 'Question cannot be blank',
   errorMissingAnswer: 'Choose a correct answer',
   errorChooseAtLeastOneCorrectAnswer: 'Choose at least one correct answer',
-  errorProvideAtLeastOneCorrectAnwer: 'Provide at least one correct answer',
-};
+  errorProvideAtLeastOneCorrectAnswer: 'Provide at least one correct answer',
+
+  // copy/clipboard strings
+  undo: 'Undo',
+  cancel: 'Cancel',
+  creatingCopies: 'Creating {count, plural,\n =1 {# copy}\n other {# copies}}...',
+  creatingClipboardCopies:
+    'Creating {count, plural,\n =1 {# copy}\n other {# copies}} on clipboard...',
+  copiedItems: 'Copied {count, plural,\n =1 {# item}\n other {# items}}',
```

Comment:
Is there a reason we're putting these strings here instead of the places they get used? In general, I would prefer to keep these associated with their respective components to make it easier to keep track of things, but am open to this if it's better practice

Reply:
Our general practice has been to have strings in situ so that we can better audit when strings are unused (especially thanks to @MisRob's awesome custom Vue component linting rules that can tell us that automatically), so if these are only being used in a single component, we should keep them there, definitely!

Reply:
I can put them in the template. I was trying to avoid duplicating these semi-intense strings.

Reply:
For these, maybe it's a matter of using something like a contentNodeActionsMixin to handle duplicate translations?